app/pages/tokens/send_reset_password.php

```diff
@@ -17,11 +17,11 @@
             ->createEmail($user->real_email, $subject, $content)
             ->send();
         if ($result->success) {
-            logger()->info("User $user->id password reset email sent");
+            logger()->info("User $user->login password reset email sent");
```

Comment:
Add login inside the context here, with PSR3 formatting
```suggestion
            logger()->info("User {login} password reset email sent", ["login" => $user->login]);
```

Reply:
✅